my-app/src/Api.js

```diff
@@ -25,26 +9,18 @@ const continueReading = (articleID) => {
   return `<p class="continue-reading"><a href="${articleID}">Continue reading...</a></p>`;
 };
 
-function Api(props) {
-  const [title, setTitle] = useState(0);
-  const [body, setBody] = useState(0);
-  const link = "https://www.theguardian.com/" + props.article;
-
-  // TODO: replace modern expressions with 19th century equivalents.
-  useEffect(() => {
-    getNews(props.article).then((n) => {
-      // extract content from the API response
-      const title = n.response.content.webTitle;
-      const bodyResponse = n.response.content.fields.body;
-      const body = props.percentage
-        ? trimArticle(bodyResponse, props.percentage) + continueReading(link)
-        : bodyResponse;
-
-      setTitle(title);
-      setBody(body);
-    });
-  }, []);
+// TODO: replace modern expressions with 19th century equivalents.
 
+function Api(props) {
+  // extract content from the API response stored in all.json
+  const articleLink = props.article.response.content.id;
+  const link = "https://www.theguardian.com/" + articleLink;
+  const title = props.article.response.content.webTitle;
+  const bodyResponse = props.article.response.content.fields.body;
+  const body = props.percentage
```

Comment:
Looks good, thanks! I've noticed that articles still show "continue reading..." if the specified percentage is set to 100. So I'd just change line 20 to the following:

```javascript 
const body = props.percentage && props.percentage !== "100"
```

Reply:
Well spotted! Have changed line 20.